cmd/fyne/internal/commands/build.go

```diff
@@ -188,21 +187,18 @@ func (b *Builder) createMetadataInitFile() (func(), error) {
 	return func() { os.Remove(metadataInitFilePath) }, err
 }
 
-func (b *Builder) injectMetadataIfPossible(runner runner, createMetadataInitFile func() (func(), error)) func() {
-	if fyneGoModVersion, err := getFyneGoModVersion(runner); err == nil {
+func (b *Builder) injectMetadataIfPossible(runner runner, createMetadataInitFile func() (func(), error)) (func(), error) {
+	fyneGoModVersion, err := getFyneGoModVersion(runner)
+	if err == nil {
```

Comment:
Go (and Fyne) prefers exit early so happy path is left aligned - so exit with nil, err in this if err != nil, the rest gets less indented.